src/service.py

```diff
@@ -18,7 +18,7 @@ async def load(self, ctx: mlservicewrapper.core.contexts.ServiceContext):
         
         if not os.path.exists(model_path):
             url = self.get_model_url(ctx)
-
+            os.mkdir(model_path.rsplit('/', 1)[0])
```

Comment:
Makes sense, _but,_ a few scenarios to think about or test:
1. How would this work if the directory already exists?
2. What if it doesn't exist, but its parent _also_ doesn't exist?
3. Would it work on Windows?

Reply:
all the three items should be fix now